Docs/HowToRunMull.md

```diff
@@ -29,11 +29,50 @@ See:
 
 ## Step 2: Getting LLVM bitcode
 
-TODO: describe at least 3 different options to obtain LLVM bitcode:
+There are at least 3 different options to obtain LLVM bitcode. Keep in mind
+that in each case you also want to add the following flags:
 
-- LTO
-- `-emit-llvm -S`
-- LTO or emit-llvm integrated to a build system
+`-g` Tells `clang` to generate a debug information which helps Mull to generate
+useful information about the code that is tested, for example the file/line
+information that is attached to mutation points.
+
+`-O0` This flag is enabled by default if not specified. However some projects
+might have exotic build systems where `-O0` must be explicitly enforced, for
+example: `s/-O3/-O0/g`, so it is a good practice to always specify this flag to make sure Mull works on unoptimized code.
+
+### -emit-llvm
+
+```bash
+$ clang -g -O0 -emit-llvm -c a-file.cpp -o a-file.bc
```

Comment:
Can we replace `a-file` with `file`? Too many dashes in this line make it hard to read.